container.go

```diff
@@ -3,6 +3,7 @@ package docker
 import (
 	"context"
 	"fmt"
+	"github.com/docker/docker/api/types/container"
 	"time"
 
 	"github.com/docker/docker/api/types"
```

Comment:
Are these supposed to be ordered?  (What is the blank line for, and why do we have `github.com` references on both sides of it?)

Reply:
That's odd. I thought the formatter would order them

Reply:
They're formatted now.